src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
-body {
+@import "./abstracts/vars";
+@import "./abstracts/mixins";
+@import "./abstracts/functions";
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  overflow-x: hidden;
+  height: 100vh;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("../images/mesh-179.png");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calendarWidth($day-size, $day-gap, $calendar-pdg);
+  gap: $day-gap;
+  padding: $calendar-pdg;
+
+  $days:
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
+
+@each $name, $size in $days {
+  &--start-day-#{$name} {
+    .calendar__day:nth-child(1) {
+      margin-left: $size;
+    }
+  }
+}
+
+  &__day {
+    @include daySize($day-size);
+    background-color: $day-bg-clr;
+    border: 1px solid $day-bdr-clr;
+    font-size: $font-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all 0.5s cubic-bezier(0.17, 0.84, 0.44, 1);
```

Comment:
fancy bezier transition 😆
Important thing - list all properties that you apply transition to. It will be easier to understand which properties are affected by it and also there will be no case in which something that should not have a transition has one